### PR TITLE
OTP26 - Get CA certificates for SSL connection

### PR DESCRIPTION
**Pull request by tuomohopia**
https://github.com/ryanwinchester/tmi.ex/issues/20

### CODE REVIEW

File: lib/tmi/client.ex [hunk 1]
```diff
@@ -84,7 +84,9 @@ defmodule TMI.Client do
   """
   @spec connect_ssl(Conn.t()) :: :ok | {:error, any()}
   def connect_ssl(%Conn{} = conn) do
-    Client.connect_ssl!(conn.client, conn.server, conn.port)
+    options = [verify: :verify_peer, cacerts: :public_key.cacerts_get()]
```
Comment (tuomohopia) [inline lib/tmi/client.ex:87]: This fails at runtime if no certs could be loaded: https://github.com/erlang/otp/blob/99e81e261096d9390fa2691d9a6f98589848c200/lib/public_key/src/pubkey_os_cacerts.erl#L42-L46

So I suppose there could be a rescue wrapper here that defaults to `options = [verify: :verify_none]` in that case.

**Comment by ryanwinchester** [comment]
Thanks.

I have this fixed in a branch I've been working on but it will be a while before I can merge that one to `main`.

**Comment by rsmidt** [comment]
Is there a chance to have land this in `main` before the big rewrite?

**Comment by ryanwinchester** [comment]
This is merged into `main` already. I don't remember if there is a release with it yet, but I am pushing a release tonight

**Comment by ryanwinchester** [comment]
`v0.7.0` pushed

**Pull request merged**